src/main/java/ca/bc/gov/backendstartapi/endpoint/CheckEndpoint.java

```diff
@@ -7,13 +7,15 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 import reactor.core.publisher.Mono;
 
 /** This class represents a check endpoint object. */
 @Slf4j
 @RestController
+@CrossOrigin(originPatterns = {"http://localhost:3000", "https://*.apps.silver.devops.gov.bc.ca"})
```

Comment:
Aren't those variables supposed to be passed as environment variables?

Reply:
Not sure if we can. Let me check!

Reply:
No, we can't. Wait! We can't here, but we can try a different approach!

Reply:
It's fine for now, we can think about that later.

Reply:
Found a way! Coming on the next commit :) @dogivalfer